tests/ops/models/test_detection_discovery.py

```diff
@@ -60,6 +60,11 @@ def create_staged_resource(self, db: Session):
             },
         )
         return resource
+    
+    def test_get_urn_list(self, db: Session, create_staged_resource) -> None:
+        urn_list = [create_staged_resource.urn]
+        from_db = StagedResource.get_urn_list(db, urn_list)
+        assert len(from_db) == len(urn_list)
```

Comment:
nitpick so feel free to ignore,  but maybe we can add an assertion like `assert from_db[0].urn == urn_list[0]` just to make sure it's returning the expected value as well :)

Reply:
added here: https://github.com/ethyca/fides/pull/5192/files#diff-8b0a478fb00bae47e3ac3533a1e7f5629cf15e077311ce6ebc7f5a4c25e3f3bfR67